cross/x86_ci_script.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function exit_with_error {
+    set +x
+
+    local errorMessage="$1"
+
+    echo "ERROR: $errorMessage"
+    exit 1
+}
+
+#Exit if input string is empty
+function exit_if_empty {
+    local inputString="$1"
+    local errorMessage="$2"
+    local printUsage=$3
+
+    if [ -z "$inputString" ]; then
+        exit_with_error "$errorMessage" $printUsage
```

Comment:
exit_with_error isn't reading the second parameter passed to it anyways. Just remove the `printUsage` stuff?